README.md

```diff
@@ -36,9 +59,27 @@ verbing('swimming')
 verbing('go')
 // =>  'go'
 ```
- 
+//Answer to 6 lines 63-74-??? Not clear
+let str = "dock"
+let atLeastThree= "ing"
+let alreadyEnding = "ly"
+
+if (str.length >= 3){
+  console.log(str + atLeastThree)
+} else {
+if (str + atLeastThree)
+  console.log(str + "ly")
+if str.length < 3,
+  console.log(str)
+}
+
```

Comment:
For this problem, working solution would look like this: 
let word = "train";
let verbing = word + "ing";
let adj = word + "ly";

if (word.slice(word.length - 3) === "ing") {
  console.log(adj);
} else if (word.length < 3) {
  console.log(word);
} else {
  console.log(verbing);
}

Try to run this code on your own and figure out how this code is working. It is important to understand what each line is doing.